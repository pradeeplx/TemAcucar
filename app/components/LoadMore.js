import React from 'react'
import Link from "./Link"

export default LoadMore = (props) => (
  <Link {...props} style={[{
    marginTop: 10,
    marginBottom: 20,
    textAlign: 'center',
  }, props.style]}>
    Carregar mais
  </Link>
)
