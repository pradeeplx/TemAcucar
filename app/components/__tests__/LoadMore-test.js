import 'react-native'
import React from 'react'
import LoadMore from '../LoadMore'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <LoadMore style={{top: 0}} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
