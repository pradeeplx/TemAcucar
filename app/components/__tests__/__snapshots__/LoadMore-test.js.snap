exports[`test renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#654C50",
          "fontFamily": "Avenir",
          "fontSize": 18,
        },
        Array [
          Object {
            "color": "#654C50",
          },
          Array [
            Object {
              "marginBottom": 20,
              "marginTop": 10,
              "textAlign": "center",
            },
            Object {
              "top": 0,
            },
          ],
        ],
      ]
    }>
    Carregar mais
  </Text>
</View>
`;
